### PR TITLE
update peaceful zen

### PR DESCRIPTION
**Pull request by fortinux**
**\- What I did**

**\- How I did it**

**\- How to verify it**

**\- Description for the changelog**

**\- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -54,6 +54,7 @@ var (
 		"naughty",
 		"nauseous",
 		"nostalgic",
+		"Peceful",
```
Comment (justincormack) [inline pkg/namesgenerator/names-generator.go]: should be a lower case `p` and spelled `peaceful`

**Comment by thaJeztah** [comment]
@fortinux can you squash and sign-off your commits?

**Comment by crosbymichael** [comment]
LGTM

**Comment by cpuguy83** [comment]
This isn't signed off yet, we can't merge.

**Comment by thaJeztah** [comment]
carried in https://github.com/docker/docker/pull/23435

**Pull request closed without merging**